autogpt/llm/providers/openai.py

```diff
@@ -27,6 +33,7 @@
         prompt_token_cost=0.0004,
         completion_token_cost=0.0,
         max_tokens=8191,
+        soft_token_limit=int(8191 * SOFT_TOKEN_LIMIT),
         embedding_dimensions=1536,
```

Comment:
i dont like feature creeping myself but should I do the same for embedding models lol? @ntindle

Reply:
Yes but not in this pr